src/tutorials/app.md

```diff
@@ -91,11 +91,11 @@ Password is `cozy`. In this mode HMR (Hot Module Replacement) is available.
 
 ## Going Further
 
-### What is `cozy.tools` ?
+### What is `cozy.localhost` ?
 
-You may see in our documentation that we use `*.cozy.tools` as domain for the local Cozy. Since Cozy uses subdomains to serve the applications, we can't use directly `localhost` as it.
+You may see in our documentation that we use `*.cozy.localhost` as domain for the local Cozy. Since Cozy uses subdomains to serve the applications, we can't use directly `localhost` as it.
 
-We have just parameterized this `*.cozy.tools` domain to always redirect to `127.0.0.1`, your local computer address.
+We have just parameterized this `*.cozy.localhost` domain to always redirect to `127.0.0.1`, your local computer address.
```

Comment:
we didn't paramtrized anything, this is normal OS behaviour to resolve naything.localhost to both 127.0.0.1 and ::1